test/bdd/features/didexchange_e2e_sdk.feature

```diff
@@ -45,3 +45,23 @@ Feature: Decentralized Identifier(DID) exchange between the agents using SDK
 
     Then   "Alice" retrieves connection record and validates that connection state is "completed"
       And   "Bob" retrieves connection record and validates that connection state is "completed"
+
+  @webkms_didexchange_e2e_sdk
+  Scenario: did exchange e2e flow with agents using webkms
+    Given "Sudesh" agent is running on "localhost" port "random" with "http" as the transport provider using webkms with key server at "https://kms.example.com:8076" URL, using "did:key:dummy-sample:sudesh" controller
```

Comment:
Use localhost:8076

Reply:
the key server returns `kms.example.com` as part of the keystore creation URL

Reply:
updated